src/main/java/nl/tudelft/pixelperfect/Game.java

```diff
@@ -170,36 +185,107 @@ public Spaceship getSpaceship() {
   @Override
   @SuppressWarnings({ "checkstyle:methodlength" })
   public void simpleUpdate(float tpf) {
-    if (moveForward) {
-      observer.move(VRApplication.getFinalObserverRotation().getRotationColumn(2).mult(tpf * 8f));
-    }
-    if (moveBackwards) {
-      observer.move(VRApplication.getFinalObserverRotation().getRotationColumn(2).mult(-tpf * 8f));
-    }
-    if (rotateLeft) {
-      observer.rotate(0, 0.75f * tpf, 0);
-    }
-    if (rotateRight) {
-      observer.rotate(0, -0.75f * tpf, 0);
-    }
+    gameState.update(tpf);
+    gameState = gameState.handleState();
+  }
 
-    scheduler.update(tpf);
-    spaceship.update(tpf);
+  /**
+   * Getter for observer.
+   * @return observer
+   */
+  public Spatial getGameObserver() {
+    return observer;
+  }
 
-    // Update the in-game heads up display.
-    gameHud.updateHud();
+  /**
+   * Getter for moveForward.
+   * @return moveForward
+   */
+  public boolean isMoveForward() {
+    return moveForward;
+  }
 
-    if (spaceship.isDead()) {
-      this.stop();
-    }
+  /**
+   * Getter for moveBackwards.
+   * @return moveBackwards
+   */
+  public boolean isMoveBackwards() {
+    return moveBackwards;
+  }
 
-    for (Event event : spaceship.getLog().getEvents()) {
-      event.notification(scene);
-    }
+  /**
+   * Getter for rotateLeft.
+   * @return rotateLeft
+   */
+  public boolean isRotateLeft() {
+    return rotateLeft;
+  }
 
-    if (spaceship.isVictorious()) {
-      System.out.println("Well played, you have completed the game!");
-      this.stop();
-    }
+  /**
+   * Getter for rotateRight.
+   * @return rotateRight
+   */
+  public boolean isRotateRight() {
+    return rotateRight;
+  }
+
+  /**
+   * Getter for startKey.
+   * @return startKey
+   */
+  public boolean isStartKey() {
```

Comment:
Both are booleans linked to keypresses, so basically "is the start key pressed", "is rotate left key pressed"